_episodes/01-shell-basics.md

```diff
@@ -69,7 +70,7 @@ Debian) providing those helpful command line utilities.
 
 In this module we will first become familiarized with basic (and
 possibly advanced) features of a shell and shell scripting, and then
-review a few aspects which particularly relate to reproducibility, as
+review a few aspects t particularly relate to reproducibility, as
```

Comment:
it is always valuable to review `git diff` or better `git diff --color-words` in this case before committing ;)